fetch/main.js

```diff
@@ -0,0 +1,30 @@
+const users = fetch('https://jsonplaceholder.typicode.com/users');
+// users is a promise, with some state and value of {Response}
+
+users
+  .then(response => response.json())
+  // then is a Promise prototype method that triggers on promise furfillment
+  // and also returns other promises that may be chained.
+  // has an optional second argument for onreject
+  .then(
+    data => {
+      console.log(data);
+    },
+    error => {
+      console.error(error);
+    }
```

Comment:
While this works to capture errors, it is better to use `.catch` instead of the second argument to `.then`

Reply:
good to know, thanks!